web/modules/custom/dbcdk_community_devel/dbcdk_community_devel.info.yml

```diff
@@ -9,3 +9,4 @@ dependencies:
   - field_ui
   - page_manager_ui
   - monolog
+  - restui
```

Comment:
Resolved but since it's not a change to line 12, this comment is still showing :)